vagrant/provision-master.sh

```diff
@@ -55,11 +55,11 @@ pushd /vagrant
       --node="${minion}" \
       --hostnames="${minion},${ip}" \
       --master="https://${MASTER_IP}:8443" \
-      --node-client-certificate-authority="${CERT_DIR}/ca/cert.crt" \
-      --certificate-authority="${CERT_DIR}/ca/cert.crt" \
-      --signer-cert="${CERT_DIR}/ca/cert.crt" \
-      --signer-key="${CERT_DIR}/ca/key.key" \
-      --signer-serial="${CERT_DIR}/ca/serial.txt"
+      --node-client-certificate-authority="${MASTER_CONFIG_DIR}/master/ca.crt" \
```

Comment:
Actually, CERT_DIR and MASTER_COFNIG_DIR are the same, and CERT_DIR is passed into `create-master-certs` above... might make sense to use it here for the cert args and just fix the filenames